### ISSUE TITLE
It would be nice if it was possible to delete the game after playing.

### ISSUE THREAD

**Issue by Kraaketaer** [wheybags/DungeonsAndDirectories#2]
Extracted this on my W10 desktop. Enjoyed playing it, but now I want to delete it. This seems entirely impossible. Apart from Windows' difficulty handling hundreds of thousands of files in a single delete operation, it gives constant errors about files not being found, cancelling the delete operation. Forcing it to skip the recycle bin allows for deletion of some files, but a hidden .game folder is left with tens of thousands of files, which is not removable through any conventional means, as it keeps throwing repeated file not found errors. As such, it would be very nice if there was an "uninstall.bat" in the .zip file.

**Comment by wheybags** [comment]
Good point, just added an unistall.bat [here](https://github.com/wheybags/DungeonsAndDirectories/commit/632378a47d508231c2ccee594d91f34f8361b811). Running the command from there should let you clean up. Thanks!

**Comment by kmanev** [comment]
The uninstall script throws the same errors (at least for me on W10).

`Deleting generated files...
.game\l1\ssstEssstFOHZcL5ElY1L\00_Set deep into the wall on the North end of the room, is a large and imposing door. - The system cannot find the file specified.
.game\l1\ssstEssstFOHZcL5ElY1L\01_It is made of thick hardwood, and revolves on a masive iron hinge. - The system cannot find the file specified.
.game\l1\ssstEssstFOHZcL5ElY1L\02_Seeping around the edges is a slight breeze of some fresher air. - The system cannot find the file specified. ...`

**Comment by wheybags** [comment]
Are you running on any kind of odd filesystem or device? I can't reproduce this, it works fine for me.

**Comment by kmanev** [comment]
I don't think I have anything odd that should relate to that issue, but to be honest for about a month now my W10 desktop has been quite unstable probably due to some buggy update... Otherwise I don't know what could be different. Hopefully I'm alone with this problem. It would be good I guess if you could share here another method for forcing the removal of the files. Cheers

**Comment by gala8y** [comment]
I contacted wheybags privately and asked him to provide here the solution I've found. He did not, so I will post it here myself.

Script created lots of files ending with '.' (dot) and without extension. Seems Windows is stupid enough to stumble and fall when encountering such a file. My solution was to use Double Commander (open source file manager) (http://doublecommander.com/) and delete files using DEL button on a keyboard (this evokes internal DC file operations handler instead of Windows handler).

HTH

**Comment by gala8y** [comment]
Script created lots of files ending with '.' (dot) and without extension. Seems Windows is stupid enough to stumble and fall when encountering such a file. My solution was to use Double Commander (open source file manager) (http://doublecommander.com/) and delete files using DEL button on a keyboard (this invokes internal DC file operations handler instead of Windows handler).

HTH

**Comment by sander1095** [comment]
I've had this issue for a while now. I'm on Windows 11. Double Commander isn't doing anything for me, nor was `rm` on windows or simply deleting the folder.

However, I've installed WSL + ubuntu and I ran `rm -rfv DUNGEONS_DIR_HERE` in the windows filesystem, and well, it's a glorious sight in my terminal so far.. ;)

I hope it finishes successfully; it's been a few minutes so far ;)

![image](https://github.com/wheybags/DungeonsAndDirectories/assets/7312681/4e8d749f-8766-4030-9c6b-a3d5db1c6388)

Update: it worked!
